packages/sommelier.rb

```diff
@@ -214,17 +273,14 @@ def self.build
               --xwayland-path=/usr/local/bin/Xwayland \
               --xwayland-gl-driver-path=#{CREW_LIB_PREFIX}/dri \
               --peer-cmd-prefix=#{CREW_PREFIX}#{@peer_cmd_prefix} \
-              --enable-xshape \
-              --noop-driver \
+              --noop-driver ${xdg_shell_v6} \
               --no-exit-with-child \
-              /bin/sh -c \\"touch ~/.Xauthority
-                xauth -f ~/.Xauthority add ${DISPLAY} . $(xxd -l 16 -p /dev/urandom)
-                source #{CREW_PREFIX}/etc/sommelierrc\\""
-            echo $sommelier_cmd >> #{CREW_PREFIX}/var/log/sommelier.log
-            $sommelier_cmd &>> #{CREW_PREFIX}/var/log/sommelier.log
+              /bin/sh -c ${xauth_cmd}"
+            echo "$sommelier_cmd" >> #{CREW_PREFIX}/var/log/sommelier.log
+            $sommelier_cmd &>> #{CREW_PREFIX}/var/log/sommelier.log &
 
-            # echo "${!}" > #{CREW_PREFIX}/var/run/sommelier-xwayland.pid
             pgrep Xwayland > #{CREW_PREFIX}/var/run/sommelier-xwayland.pid
```

Comment:
Add echo to display with `-v`?

Reply:
This is already echoed to the console:
```
Sommelier can use direct scaling.
Sommelier SCALE automatically set to .5.
SCALE may be manually set in ~/.sommelier.env .
sommelier processes running:  4658
```

Reply:
The command output is printed but the echo is to display the command that was executed.